.cirrus.yml

```
@@ -150,12 +148,15 @@ non_standard_toolchains_task:
         - make -j$(nproc) -sk
       tests_script:
         - >
-          for i in cpu disk interface load memory swap users; do
-            if ! $(ldd ".libs/${i}.so" 2>/dev/null | grep -q 'libstatgrab.so'); then
-              echo "plugin $i NOT linked against libstatgrab"
-              exit 1
+          err=0
+          for plugin in cpu disk interface load memory swap users; do
+            if ! $(ldd ".libs/${plugin}.so" 2>/dev/null | grep -q 'libstatgrab.so'); then
+              echo "plugin ${plugin} is NOT linked against libstatgrab:"
+              ldd ".libs/${plugin}.so" | sed -e 's/^/    /'
+              err=$(($err + 1))
             fi
           done
+          exit $err
```

Because of this error:
```
err=0 for plugin in cpu disk interface load memory swap users; do
/tmp/scriptsb51677938757ea6f822ffb83f91a272f.sh: line 5: syntax error near unexpected token `do'
```

I think this needs semicolons to separate statements:
```suggestion
          err=0;
          for plugin in cpu disk interface load memory swap users; do
            if ! $(ldd ".libs/${plugin}.so" 2>/dev/null | grep -q 'libstatgrab.so'); then
              echo "plugin ${plugin} is NOT linked against libstatgrab:";
              ldd ".libs/${plugin}.so" | sed -e 's/^/    /';
              err=$((err + 1));
            fi;
          done;
          exit $err
```
(And `$` is redundant for variable names within `$(())`, as `shellcheck` likes to often remind me.)

---

I think the problem is the `>` multi-line indicator, because it replaces newlines with spaces. With `|` this should work. Now I don't know how this could ever have worked previously.